src/components/LeaderBoardHeader/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from "react";
+import TopUser from "./TopUser";
+import headerImage from "../../img/svg/headerImage.svg"
+
+import styles from "./LeaderBoardHeader.module.scss";
+
+const LeaderBoardHeader: FC = () => {
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.wrapper__header}>
+        Cube<span>19</span> Leaderboard
+      </div>
+      <div className={styles.wrapper__table}>
```

Comment:
Should be classname table without wrapper

Reply:
> Should be classname table without wrapper

Everywhere?